Dockerfile

```diff
@@ -6,22 +6,22 @@ FROM ${OS_NAME}:${OS_VERSION} as frr-builder
 ARG FRR_TAG
 ARG RTR_TAG
 ARG LIBYANG_VERSION
-ARG LIBYANG_BUILD
 ARG LIBYANG_BUILD_ID
+ARG LIBYANG_DISTRIBUTION
 
-ENV LIBYANG_URL=https://ci1.netdef.org/artifact/LIBYANG-YANGRELEASE/shared \
+ENV LIBYANG_URL=https://ci1.netdef.org/browse/LIBYANG-LY1REL-4/artifact/shared/${LIBYANG_DISTRIBUTION}-x86_64-Packages \
```

Comment:
Would be clearer if `4` is externalised as own ENV because it is the bamboo CI-Build number of libyang

Reply:
Yes will do

Reply:
Problem is which version to choose:
https://ci1.netdef.org/browse/LIBYANG-LY1REL

Reply:
took the latest build now